tests/chocolatey-tests/choco-info.Tests.ps1

```diff
@@ -142,4 +142,21 @@ Describe "choco info" -Tag Chocolatey, InfoCommand {
             $Output.Lines | Should -Contain "${Title}: $Value"
         }
     }
+
+    Context "Listing package information about local side by side installed package" {

```

Comment:
Could we skip this if version is below `1.2.0-alpha`? That will allow these tests to be bypassed with current stable builds of Chocolatey.

Reply:
The question would be, do we actually use the latest codebase tests when running against older versions of Chocolatey CLI?

If we are, then sure I'll add some skipping, but if not I would say that we don't need it.

Reply:
As we talked about privately, I have decided not to add the skip parameter.

I'll resolve this requested change and the other requested changes of similar nature.